src/ice_model.F90

```diff
@@ -659,7 +661,7 @@ subroutine set_ocean_top_fluxes(Ice, IST, IOF, FIA, OSS, G, US, IG, sCS)
   if (allocated(IOF%melt_nudge)) then
     do j=jsc,jec ; do i=isc,iec
       i2 = i+i_off ; j2 = j+j_off! Use these to correct for indexing differences.
-      Ice%lprec(i2,j2) = Ice%lprec(i2,j2) + US%RZ_T_to_kg_m2s*IOF%melt_nudge(i,j)
+      Ice%seaice_melt(i2,j2) = Ice%seaice_melt(i2,j2) + US%RZ_T_to_kg_m2s*IOF%melt_nudge(i,j)
```

Comment:
@theresa-morrison, I realized I don't really know what `melt_nudge` is for, but I assumed this correction should be moved from `lprec` to the new `seaice_melt` diagnostic?

Reply:
I would say so. It looks like melt associated with artificially nudging the sea ice concentration to match observations. See ~line 713 of SIS_slow_thermo. It should be zero in the Baltic test cases.